pkg/cmd/roachtest/tests/validate_system_schema_after_version_upgrade.go

```diff
@@ -118,14 +118,28 @@ func runValidateSystemSchemaAfterVersionUpgrade(
 
 	systemComparison := newTenantSystemSchemaComparison(install.SystemInterfaceName)
 	var tenantComparison *tenantSystemSchemaComparison
+	var deploymentMode mixedversion.DeploymentMode
 
-	mvt := mixedversion.NewTest(ctx, t, t.L(), c, c.All(),
+	opts := []mixedversion.CustomOption{
 		// We limit the number of upgrades since the test is not expected to work
 		// on versions older than 22.2.
 		mixedversion.MaxUpgrades(3),
 		// Fixtures are generated on a version that's too old for this test.
 		mixedversion.NeverUseFixtures,
-	)
+	}
+
+	if c.IsLocal() {
+		opts = append(opts,
+			// Separate-process deployments are still in its early
+			// days. Disable it in local clusters (such as in CI) to avoid
+			// disruptions.
+			mixedversion.EnabledDeploymentModes(
```

Comment:
Maybe drop a `TODO` so it's easier to find.

Reply:
Done.